### ISSUE TITLE
Issue on page /docs/03-images.html

### ISSUE THREAD

**Issue by jccaicedo** [cytomining/DeepProfiler-handbook#12]
This page is looking good! We need to expand its content in the following way:

* Show an example image (from the example data) with the 5 channels separated and named. The dimensions should also be illustrated. Create this image in the Google Slides and export it as a PNG. @yhan8 Can you create this example figure?
* Add an image of how the outlines look like, and more instructions on how to obtain them. This is part of the CellProfiler segmentation pipeline, we need to clarify these instructions. I believe @rsenft1 knows more details about this?

**Comment by rsenft1** [comment]
Yes, I do have a question about those outlines. Does DeepProfiler assume the outline is on the inside or outside of the cell? In other words, is the outline itself 'part' of the cell? I ask because the default with CellProfiler is to display outlines on the 'inside' but depending on how this is treated, we may want to ensure users select the 'outside' option. Here are the options from CellProfiler's help doc: 

Specify how to mark the boundaries around an object: 

* Inner: outline the pixels just inside of objects, leaving background pixels untouched. 
* Outer: outline pixels in the background around object boundaries. When two objects touch, their boundary is also marked. 
* Thick: any pixel not completely surrounded by pixels of the same label is marked as a boundary. This results in boundaries that are 2 pixels thick. 

I can help generate an example if we still need one. Can someone link me to a single image of a cell from the example dataset (or attach it here)?

**Comment by Arkkienkeli** [comment]
@rsenft1 There is an example in [training (export) section](https://cytomining.github.io/DeepProfiler-handbook/docs/07-train.html#export-single-cells) (Figure 3). In the perfect case scenario, we expect all mask pixels to belong to the object only (I believe from the description of CellProfiler outlining it is filled inner outline) because DeepProfiler simply multiplies image and binary mask.

**Comment by rsenft1** [comment]
Wait, sorry do you need an outline or a mask? Should the cells and nuclei just be outlined in white or should the pixels belonging to cells be all white? The process for going about getting outlines/masks in cellprofiler is somewhat different.

**Comment by Arkkienkeli** [comment]
Oh sorry, I got confused with single-cells. 
So in case of outlines (in `outlines` folder ), only cell outlines are needed. 

About Inner \ Outer \ Thick - it becomes a trickier question: @jccaicedo could you please help?

**Comment by jccaicedo** [comment]
I believe the default option with CellProfiler should be good, as we have traditionally followed the Cell Painting pipeline.
@rsenft1 DeepProfiler needs outlines as an input, but it transforms those outlines to masks. Thus, no need to save the masks. Does this make sense?

**Comment by jccaicedo** [comment]
@Arkkienkeli You could select an example image from one of our datasets and display both the original (a pseudo-color composite) and the outlines. This is necessary to give people an idea of what they should aim for.

@rsenft1 if you can point to CellProfiler instructions on how to get these images, that'd be great!

**Comment by rsenft1** [comment]
Got it. Just to make sure I understand, how does DeepProfiler handle two different cell outlines that touch? Conversely, what about a cell that narrows to a point and then expands again (like the axon of a neuron might get very thin and then slightly thicker? 

Usually, the way we'd deal with including information about where cells are is to use a label matix where pixels belonging to each cell are all of a given value (cell 1 pixels are all 1, cell 2 pixels are all 2 etc.) But with the outlines, it seems like there's some room for ambiguity.

**Comment by rsenft1** [comment]
@jccaicedo could I get your thoughts on ⬆️ ? I'm happy to write the section, but I'd definitely want to include some info on these edge cases as well.
